src/ScriptEditor/NetscriptDefinitions.d.ts

```diff
@@ -4325,11 +4325,20 @@ export interface InfiltrationReward {
  * @public
  */
 export interface InfiltrationLocation {
-  location: any;
+  location: SimpleLocation;
   reward: InfiltrationReward;
   difficulty: number;
 }
 
+/**
+ * Infiltration API.
+ * @public
+ */
+interface SimpleLocation {
+  city: CityName | null;
+  name: LocationName;
+}
```

Comment:
I'm not sure what the best practice is here. These need to be imported in order to be resolved [^1], but they're `enum` with string values, so do they belong in a `.d.ts` file?

[^1]: As does `Location` from my suggestion above